plugins/automox/icon_automox/actions/get_device_by_hostname/action.py

```diff
@@ -0,0 +1,19 @@
+import insightconnect_plugin_runtime
+from .schema import GetDeviceByHostnameInput, GetDeviceByHostnameOutput, Input, Output, Component
+# Custom imports below
+
+
+class GetDeviceByHostname(insightconnect_plugin_runtime.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='get_device_by_hostname',
+                description=Component.DESCRIPTION,
+                input=GetDeviceByHostnameInput(),
+                output=GetDeviceByHostnameOutput())
+
+    def run(self, params={}):
+        device = self.connection.automox_api.find_device_by_attribute(params.get(Input.ORG_ID),
+                                                                      ["name"],
```

Comment:
is the extra "," a typo or apart of convention?

Reply:
Not sure I follow. There should be three parameters: org id, list of attribute values, and the actual value. Perhaps I'm overlooking the extra ","